index.html

```diff
@@ -796,10 +811,7 @@ <h2>Attributes</h2>
               by the operating system or network hardware. On getting, it
               <em class="rfc2119" title="MUST">MUST</em> return the value of the
               <code><a>RTCQuicStream</a></code>'s <a>[[\WriteBufferedAmount]]</a> internal slot.
-              If the <code><a>RTCQuicStream</a></code> is in the <code>closed</code>

```

Comment:
I took this out (for the writeBufferedAmount). This isn't applicable, because the closed state means the stream isn't writable, so increasing it isn't possible.

Reply:
Makes sense.